src/main/kotlin/io/sip3/salto/ce/sip/SipRegisterHandler.kt

```diff
@@ -98,8 +98,12 @@ open class SipRegisterHandler : AbstractVerticle() {
         }
 
         vertx.setPeriodic(expirationDelay) {
-            terminateExpiredRegistrations()
-            terminateExpiredSessions()
+            try {
```

Comment:
Let's better add this `try-catch` as a part of `sip3-commons` setPeriodic method()